adagrams/game.py

```diff
@@ -1,11 +1,104 @@
+import random
+from re import A
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    letters = []
+    
+    for letter in LETTER_POOL:
+        for num in range(LETTER_POOL[letter]):
+            letters.append(letter)
+    
+
+    hand = []
+
+    while len(hand) < 10:
+        rand_letter = random.choice(letters)
+        hand.append(rand_letter)
+        letters.remove(rand_letter)
+    return hand 
 
 def uses_available_letters(word, letter_bank):
-    pass
+    available_letters = []
+
+    for letter in letter_bank:
+        letter_capitalize = letter.capitalize()
+        available_letters.append(letter_capitalize)
+
+    for letter in word:
+        letter_capitalize = letter.capitalize()
+        if letter_capitalize in available_letters:
+            available_letters.remove(letter_capitalize)
+        else:
+            return False
+    return True
```

Comment:
Great logic! One small note: though both work in this case, it would be slightly clearer to use `upper` instead of `capitalize` here. `capitalize` is used to just capitalize the first letter of a string, while `upper` capitalizes all letters in a string. Because you're only dealing with single-letter strings, they are equivalent, but we would slightly prefer the simpler `upper` method.